newIDE/app/src/MainFrame/index.js

```diff
@@ -1159,7 +1158,9 @@ const MainFrame = (props: Props) => {
     getStorageProviderOperations,
     afterCreatingProject: async ({ project, editorTabs, oldProjectId }) => {
       setNewProjectSetupDialogOpen(false);
-      await closeExampleStoreDialog();
+      onOpenExampleStoreDialog(false);
+      onSelectExampleShortHeader(null);
+      onSelectPrivateGameTemplateListingData(null);
```

Comment:
should those 3 lines be coming from the hook as a function `closeExampleStoreDialog`?
This could allow hiding that logic inside?
And avoid those weird named function `open(false)`